src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/repository/Node.kt

```diff
@@ -0,0 +1,210 @@
+package com.cognifide.gradle.aem.common.instance.service.repository
+
+import com.cognifide.gradle.aem.common.http.RequestException
+import com.cognifide.gradle.aem.common.utils.Formats
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.jayway.jsonpath.PathNotFoundException
+import java.io.Serializable
+import java.util.*
+import net.minidev.json.JSONArray
+import org.apache.jackrabbit.vault.util.JcrConstants
+
+class Node(private val repository: Repository, val path: String) : Serializable {
```

Comment:
clue of API :)

Reply:
I wonder if https://github.com/Kotlin/kotlinx.serialization#quick-example is applicable here for serialization.